tdrs-backend/tdpservice/search_indexes/admin/tanf.py

```diff
@@ -1,9 +1,9 @@
 """ModelAdmin classes for parsed TANF data files."""
-from django.contrib import admin
-from .filter import CreationDateFilter
+from .filters import CreationDateFilter, FiscalPeriodFilter, STTFilter
+from .mixins import CsvExportAdminMixin, ReadOnlyAdminMixin
 
 
-class TANF_T1Admin(admin.ModelAdmin):
+class TANF_T1Admin(ReadOnlyAdminMixin, CsvExportAdminMixin):
     """ModelAdmin class for parsed T1 data files."""
 
     list_display = [
```

Comment:
case_number, county fips, zip code, and stratum can be dropped from the list display in DAC

![2915p1](https://github.com/raft-tech/TANF-app/assets/63075587/882db073-6977-451b-bf62-643a949b1697)